pkg/registry/authentication/tokenreview/storage.go

```diff
@@ -68,14 +76,24 @@ func (r *REST) Create(ctx context.Context, obj runtime.Object, createValidation
 	fakeReq := &http.Request{Header: http.Header{}}
 	fakeReq.Header.Add("Authorization", "Bearer "+tokenReview.Spec.Token)
 
+	auds := tokenReview.Spec.Audiences
+	if len(auds) == 0 {
+		auds = r.apiAudiences
+	}
+	if len(auds) == 0 {
```

Comment:
if len >= 0, right? (and if tests passed with this, add a test that would have caught this)

Reply:
Nice catch. This area is covered by integration tests which failed. https://gubernator.k8s.io/build/kubernetes-jenkins/pr-logs/pull/62692/pull-kubernetes-integration/36093/